operator/controller/src/test/deploy/catalog/catalog.template.yaml

```diff
@@ -0,0 +1,13 @@
+schema: olm.template.basic
+entries:
+  - schema: olm.package
+    name: apicurio-registry
+    defaultChannel: 3.x
+  - schema: olm.channel
+    package: apicurio-registry
+    name: 3.x
+    entries:
+      - name: apicurio-registry.v${PLACEHOLDER_LC_VERSION}
+      # TODO: Replaces
```

Comment:
is the `TODO` still relevant?

Reply:
yes, we need to handle this when we have multiple CSVs.

Reply:
right :+1:

Reply:
:+1: